docs/html/reference/pip_install.rst

```diff
@@ -913,8 +920,9 @@ Examples
 
     ::
 
-      $ pip install SomeProject==1.0.4@http://my.package.repo//SomeProject-1.2.3-py33-none-any.whl
-      $ pip install "SomeProject==1.0.4 @ http://my.package.repo//SomeProject-1.2.3-py33-none-any.whl"
+      $ pip install SomeProject@http://my.package.repo//SomeProject-1.2.3-py33-none-any.whl
+      $ pip install "SomeProject @ http://my.package.repo//SomeProject-1.2.3-py33-none-any.whl"
```

Comment:
It's unclear from https://github.com/pypa/pip/pull/6203/files why there are two `/` here (and I see no reason why) so:

```suggestion
      $ pip install SomeProject@http://my.package.repo/SomeProject-1.2.3-py33-none-any.whl
      $ pip install "SomeProject @ http://my.package.repo/SomeProject-1.2.3-py33-none-any.whl"
```

Reply:
fixed.